backend/modules/pdf_processing/parser_service.py

```
@@ -0,0 +1,226 @@
+                        if session_id and progress_manager:
+                            progress_manager.error_processing(session_id, "No content extracted from PDF")
+                        raise HTTPException(status_code=500, detail="No content could be extracted from the PDF")
+                        
+                finally:
+                    # Clean up temporary file
+                    try:
+                        os.unlink(temp_file_path)
+                    except Exception as e:
+                        debug_log(f"[LLAMAPARSE] Warning: Could not delete temp file {temp_file_path}: {e}")
+                        
+            except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)